src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -305,14 +300,8 @@ Command HttpServer()
                 httpCommand.Handler = CommandHandler.Create<StartupOptions, KernelHttpOptions, IConsole, InvocationContext>(
```

Comment:
I just noticed that `KernelHttpOptions` has nothing to do with HTTP. Does it need to exist and why would it be different from the already-existing `DefaultKernel` concept?